libs/cardano-ledger-core/src/Cardano/Ledger/DRepDistr.hs

```diff
@@ -51,7 +51,7 @@ data DRepState c = DRepState
   { drepExpiry :: !EpochNo
   , drepAnchor :: !(StrictMaybe (Anchor c))
   }
-  deriving (Show, Eq, Generic)
+  deriving (Show, Eq, Ord, Generic)
```

Comment:
This is automatically using the `drepExpiry`  instance?

Reply:
Yep, this is regular stock deriving mechanism for Ord, just as it is done with Eq.